src/extendables/UserExtendables.ts

```diff
@@ -117,6 +117,28 @@ export default class extends Extendable {
 		return this.settings.update(UserSettings.QP, newQP);
 	}
 
+	public async addSlayerPoints(this: User, amount: number) {
```

Comment:
Move all of these that you added to a new file, like I made Gear and Bank ones, make a Slayer one